src/Components/DashboardMenu/DashboardMenu.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { NavLink, useLocation } from "react-router-dom";
+
+
+const DashboardMenu = () => {
+  return(
+    <div>
+      <NavLink to='/dashboard'><button>My Books</button></NavLink>
+      <NavLink to='/dashboard/my-borrowed-books'><button>My borrowed books</button></NavLink>
+      <NavLink to='/dashboard/pending-requests'><button>Pending Requests</button></NavLink>
```

Comment:
The dashboard menu disappears when I'm on pending requests

Reply:
Good catch. Adding it to `line 41` in `App.tsx` which is getting to be kind of excessive. Do you know how to conditionally render an element on multiple routes?